doc/extensions/index.rst

```diff
@@ -41,3 +41,4 @@ features by developing your own CKAN extensions.
    translating-extensions
    flask-migration
    signals
+   htmx
```

Comment:
I'd probably move this page to the Theming section, maybe after the Customizing CKAN's javascript page (which probably needs to be reviewed to mention the new htmx patterns)

Reply:
Done!